src/main/java/seedu/notus/ui/Formatter.java

```diff
@@ -51,7 +51,7 @@
      * Maximum length of a note's content to display.
      */
     private static final int CONTENT_CUTOFF = MAX_MESSAGE_LENGTH - 50;
-    private static final int CONTENT_INDENT = 3;
+    private static final int CONTENT_INDENT = 0;
```

Comment:
Can remove it since its not used anymore right?

```suggestion
```

Reply:
Could have but I just keep it, in case we need it. Can remove it during the code cleanup if confirm not needed